.gitignore

```diff
@@ -214,3 +214,4 @@ pip-log.txt
 
 #Mr Developer
```

Comment:
Please delete this entire `.gitignore` file, as our other extensions don't have one, and this repo's has a big one. Instead, add these "should be included from all my projects everywhere" type of things to your own personal computer's *global* gitignore. Instructions are here: https://help.github.com/en/github/using-git/ignoring-files